### ISSUE TITLE
Creates empty JSON files

### ISSUE THREAD

**Issue by ErikThiart** [highcharts/highcharts#6559]
#### Expected behaviour
For it to work

#### Actual behaviour
It does not work

Does not seem to work, it simply creates empty json files, I am using Lubuntu + LAMP

I am referring to the updatejson.php file -> master/samples/data/updatejson.php

**Comment by sebastianbochan** [comment]
Could you elaborate your question, its a little bit unclear.

**Comment by ErikThiart** [comment]
No problem, I am experimenting so I downloaded the file updatejson.php (master/samples/data/updatejson.php)

I then ran the file in the brochure, it proceeded to create the various json files, however those files are all empty.

I just now ran some debugging in PHP and this is the error I receive:
file(http://finance.google.com/finance/historical?q=AAPL&histperiod=daily&startdate=Apr+5,+2010&enddate=Apr+3,+2017&output=csv): failed to open stream: Network is unreachable in /var/www/html/index.php

However if I copy that URL into my browser the CSV downloads without issue.

Any idea why it won't download using PHP?
